policy/modules/services/container.if

```diff
@@ -1079,6 +1091,119 @@ interface(`container_manage_home_data_sock_files',`
 	manage_sock_files_pattern($1, container_data_home_t, container_data_home_t)
 ')
 
+########################################
+## <summary>
+##	Administrate all container files.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`container_admin_all_files',`
```

Comment:
If the interface is `admin`, then it should be using `admin_pattern`.

Reply:
OK, I've reworked most of the `admin` interfaces to use `admin_pattern` but renamed the home content access interface to `manage` because it's more general than just typical admin access.